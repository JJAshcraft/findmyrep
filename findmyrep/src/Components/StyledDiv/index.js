import styled from "styled-components";

const StyledDiv = styled.div`
  width: 100%;
  display: flex;
  align-items: center;
  margin-bottom: 10px;
  border: none;
  color: grey;
  background-color: #f8f8f9;
  border-radius: 5px;
  font-size: 16px;
  font-weight: 500;
  font-family: "Montserrat", sans-serif;
  height: 45px;
  padding: 0 5px 0 5px;
  overflow-x: auto;
  overflow-y: hidden;
  p {
    white-space: nowrap;
    vertical-align: middle;
  }
`;

export default StyledDiv;
