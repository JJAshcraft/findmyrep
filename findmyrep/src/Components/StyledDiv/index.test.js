import React from "react";
import renderer from "react-test-renderer";
import "jest-styled-components";
import StyledDiv from "./index.js";

test("StyledDiv renders with proper styling", () => {
  const tree = renderer.create(<StyledDiv />).toJSON();
  expect(tree).toMatchSnapshot();
});
