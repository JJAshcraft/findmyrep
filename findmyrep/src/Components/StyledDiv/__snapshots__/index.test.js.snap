// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledDiv renders with proper styling 1`] = `
.c0 {
  width: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  margin-bottom: 10px;
  border: none;
  color: grey;
  background-color: #f8f8f9;
  border-radius: 5px;
  font-size: 16px;
  font-weight: 500;
  font-family: "Montserrat",sans-serif;
  height: 45px;
  padding: 0 5px 0 5px;
  overflow-x: auto;
  overflow-y: hidden;
}

.c0 p {
  white-space: nowrap;
  vertical-align: middle;
}

<div
  className="c0"
/>
`;
